### ISSUE TITLE
Analysing the data

### ISSUE THREAD

**Issue by ndreey** [ndreey/01_clean_mock_data#70]
# Quick overview / initial idea
### Correlation
Determine if there exists a correlation with HC and metric.
1. Determine if data is parametric or non-parametric.
2. Do a correlation test.
3. Do a significance test.
#### Non-parametric
- Spearman's rank correlation
- Wilcoxon rank-sum test or Mann-Whitney U test
#### Parametric
- Pearsons correlation
- t-test 

### Host depletion by bin removal
- Benchmark (worst-possible situation, extreme neg value).
- Remove all host bins to create a best-possible situation (extreme pos value).
    - Remove the contigs belonging to the host genome. in the GSA.
- _Do the binning to get an estimate of a real-life situation?_

**Comment by ndreey** [comment]
# Assembly

### Num mismatches per 100kbp
There were no mismatches for the genomes of the endophytes , only in the host genome. Indicating that for the gsa of the endophytes, hc levels do not correlate with less or more mismatches.

As there is no host genome in the 00 samples, the 0.0 value for the host genome should be excluded from the correlation calculation. Therefore, I will use sample `00:095`.

To determine if the data is parametric or not I used the **Shapiro-Wilk** **test** as sample size is low (_n=10, n < 50_)
`shapiro.test(host_mm)   # W = 0.91374, p-value = 0.3077`
The test concluded that there is not sufficient evidence to reject the null hypothesis that the data is normally distributed. Therefore, I will use a non-parametric test to determine if there exists a statistical significant correlation between _number of mismatches per 100kbp_ and increased HC levels.

**Kendal tau** seems to be more advocated than **Spearman** but i will have to go in deeper to answer why. 

_Kendall tau_
```
> cor.test(hc[2:11], host_mm, method ="kendall", exact = FALSE)

	Kendall's rank correlation tau

data:  hc[2:11] and host_mm
z = -2.5145, p-value = 0.01192
alternative hypothesis: true tau is not equal to 0
sample estimates:
       tau 
-0.6292532 
```
_Spearmans_
```
> cor.test(hc[2:11], host_mm, method ="spearman", exact = FALSE)

	Spearman's rank correlation rho

data:  hc[2:11] and host_mm
S = 283.36, p-value = 0.01953
alternative hypothesis: true rho is not equal to 0
sample estimates:
       rho 
-0.7173285 
```
I used exact = FALSE as i was getting `Cannot compute exact p-value with ties`, have to read more to explain this.

In summary, both **Kendall** and **Spearman** show that the number of mismatches have a statistically significant negative correlation (_p = 0.05_, r = -(0.6~0.7) with higher HC levels. This is understandable as we get more host genome data with higher HC.

**Comment by ndreey** [comment]
# Evaluating if filtered have a difference.
Firstly i have to assemble the raw and filtered reads.
Then run them through metaQUAST.
At the moment, we have:
- GSA
- Raw
- 1000c_filt
- 700c_filt
- max_filt

Ideally, we maybe can find that the correlations are opposite to GSA

**Comment by ndreey** [comment]
# Bins to compare
I need to run concoct to generate bins that i can run through _BUSCO(??)_ to see if there is a difference. 

### CONCOCT contig length threshold
Here we see the distribution of contig lengths. x00 is the raw sample that all will use as it has not been tampered with.
![image](https://github.com/ndreey/01_clean_mock_data/assets/72344166/4babd0f6-39f8-47a2-b818-e08a84f195b6)

**Comment by ndreey** [comment]
# Assembly plots
Data analysis of gsa, raw, x1000c, x700c, and maxbin.
## Number of mismatches per 100kbp
![image](https://github.com/ndreey/01_clean_mock_data/assets/72344166/ae50e986-eb5c-490f-98af-429c96918de7)
